web/templates/stream_settings/new_stream_user.hbs

```diff
@@ -1,13 +1,17 @@
 <tr>
     <td class="panel_user_list">
-        {{> ../user_display_only_pill display_value=full_name}}
+        {{> ../user_display_only_pill display_value=full_name strikethrough=soft_removed}}
     </td>
     {{#if email}}
-        <td class="subscriber-email">{{email}}</td>
+        <td class="subscriber-email {{#if soft_removed}} strikethrough {{/if}}">{{email}}</td>
     {{else}}
-        <td class="hidden-subscriber-email">{{t "(hidden)"}}</td>
+        <td class="hidden-subscriber-email {{#if soft_removed}} strikethrough {{/if}}">{{t "(hidden)"}}</td>
     {{/if}}
     <td>
-        <button {{#if disabled}} disabled="disabled"{{/if}} data-user-id="{{user_id}}" class="remove_potential_subscriber button small rounded white">{{t 'Remove' }}</button>
+        {{#if soft_removed}}
+            <button data-user-id="{{user_id}}" class="undo_soft_removed_potential_subscriber button small rounded white">{{t 'Add' }}</button>
+        {{else}}
+            <button {{#if disabled}} disabled="disabled"{{/if}} data-user-id="{{user_id}}" class="remove_potential_subscriber button small rounded white">{{t 'Remove' }}</button>
```

Comment:
Will `soft_remove_potential_subscriber` be better here to match the class name for add button?

Reply:
One problem here is that the same template is getting used by `user_group_create_members.ts` too. Having `soft_remove_potential_subscriber` there might be more confusing than it being name `remove_potential_subscriber` for stream_create

Reply:
Ok, the current class name seems fine then.